simulator/src/main/java/com/hazelcast/simulator/worker/WorkerMessageProcessor.java

```diff
@@ -92,22 +91,18 @@ private boolean isMaster() {
                 public Boolean call() throws Exception {
                     return isOldestMember();
                 }
-            }, 10, TimeUnit.SECONDS).get(TIMEOUT, TimeUnit.SECONDS);
-        } catch (InterruptedException e) {
-            throw new IllegalStateException(e);
-        } catch (ExecutionException e) {
-            throw new IllegalStateException(e);
-        } catch (TimeoutException e) {
+            }, delaySeconds, TimeUnit.SECONDS).get(TIMEOUT_SECONDS, TimeUnit.SECONDS);
+        } catch (Exception e) {
             throw new IllegalStateException(e);
         }
     }
 
-    private boolean isOldestMember() {
+    boolean isOldestMember() {
```

Comment:
is it visibility increased due tests? that's usually considered as a dodgy practice.